### DISCUSSION TITLE
Off-centred Wallpaper

### DISCUSSION THREAD

**Discussion by Soham52** [rocksdanister/lively#1068] [General]
Bug:

The Waves Pattern in the "The Hill" Wallpaper is off-centred
 
Expected behaviour:

The wave pattern should be centred

Screenshots:

![image](https://user-images.githubusercontent.com/73898737/154648531-b303f6f9-d151-42ff-933a-820f982640e3.png)

 - OS: [e.g. Windows 10 1909]
- The Hill

Log file:

[lively_log_20220218_135304.zip](https://github.com/rocksdanister/lively/files/8095533/lively_log_20220218_135304.zip)

**Comment by dabjulmaros** [comment]
I cant seem to replicate the same behavior.
In my machine is off by maybe a pixel or 2 and I think is due to the font.
![image](https://user-images.githubusercontent.com/33079720/154777302-74407f97-ac96-4e77-a2df-47cd19e591ae.png)
1920 width
![image](https://user-images.githubusercontent.com/33079720/154777425-965bbd0a-8c29-428d-b703-183c51ecd5f0.png)|
2560 width
![image](https://user-images.githubusercontent.com/33079720/154777673-28103687-492c-4084-a006-5788e8fd56d1.png)

**Reply by Soham52** [reply]
My issue was actually not with the off-centred clock as I am using rainmeter to substitute for it, but it was for the off centred wave pattern below. But after a few tests i did conclude that the wave pattern was actually not off-centred; for me i think its looks a bit off-centred due to the waves not being symmetrical

you can see the comparison here:

https://user-images.githubusercontent.com/73898737/154784648-5204e4d3-b85a-4a64-9992-bf327962dd00.mp4
